sdlib/d/gc/util.d

```diff
@@ -46,6 +46,10 @@ unittest log2floor {
 	assert(log2floor(32) == 5);
 }
 
+ushort swapEndian(ushort x) {
+	return ushort.max & ((x >>> 8) | ((x << 8)));
```

Comment:
You don't need to use `>>>` because the type is unsigned.

Reply:
Also, why does it come in between pow2 and log2, two obviously related concepts.